check_vitals.py

```diff
@@ -0,0 +1,43 @@
+
+from time import sleep
+import sys
+
+class check_vitals:
+  def is_temperature_ok(self, temperature):
+    if temperature in range(95, 102):
+      self.print_wait_time()
+      print('Temperature is normal')
+      return True
+    else:
+      self.print_wait_time()
+      print('Temperature critical!')
+      return False
+
+  def is_pulse_rate_ok(self, pulseRate):
+    if pulseRate in range(60, 100):
```

Comment:
The pattern of code in temperature and pulse rate is same, though the tool isn't intelligent enough to catch it.
Try to remove the duplicated patterns. Hint: you can have a function like `is_parameter_ok(input_value, lower_limit, upper_limit)`